build error in usddeck.c (array subscript has type)

@whoenig : When repo is cloned as instructed, there is on error respond to make command such as:

src/deck/drivers/src/usddeck.c: In function 'f_gets_without_comments':
src/deck/drivers/src/usddeck.c:238:5: error: array subscript has type 'char' [-Werror=char-subscripts]
     if (isspace(c)) {
     ^
cc1: all warnings being treated as errors
tools/make/targets.mk:26: recipe for target 'usddeck.o' failed
make[1]: *** [usddeck.o] Error 1
Makefile:327: recipe for target 'build' failed
make: *** [build] Error 2

It has nothing about this issue, this is for micro sd deck I guess, since I am not using this deck, by commenting out problematic section, I am able to make the project and fly.

_Originally posted by @tugayalperen in https://github.com/bitcraze/crazyflie-firmware/issues/409#issuecomment-477983043_

--- a/src/deck/drivers/src/usddeck.c
+++ b/src/deck/drivers/src/usddeck.c
@@ -235,7 +235,7 @@ TCHAR* f_gets_without_comments (
     if (c == '\n') {
       break;   /* Break on EOL */
     }
-    if (isspace(c)) {
+    if (isspace((int)c)) {
       continue; /* Strip whitespace */
     }
     if (c == '#') {
@@ -338,7 +338,7 @@ static void usdInit(DeckInfo *info)
         success = true;
         break;
       }
-      
+
       if (!success) {
           DEBUG_PRINT("Config read [FAIL].\n");
       }
@@ -379,7 +379,7 @@ static void usdLogTask(void* prm)
         line = f_gets_without_comments(readBuffer, sizeof(readBuffer), &logFile);
         if (!line) break;
       }
-      
+
       while (line) {
         line = f_gets_without_comments(readBuffer, sizeof(readBuffer), &logFile);
         if (!line) break;

